mercury/templates/simulator.html

```diff
@@ -15,7 +15,24 @@
         <div id="grafana-base">
             <div align="center">
                 <h2>Telemetry Data Simulator</h2>
-                <br>
+                <br>  <div class="sim-panel"> <b> Welcome to our Telemetry Data Simulator. </b><br> <br> 
+
+                    <input class="submitbutton_temp simulator-btn grafana-btn-green" name="Continuous" value="Continuous Submission"> 
+                    or <input class="submitbutton_ss simulator-btn grafana-btn-red" name="Once" value="Submit Once">: will generate data for individual panels <br> <br> 
+                    <input class="submitbutton_temp simulator-btn grafana-btn-green" name="Continuous" value="Continuous Submission">: repeatedly generates data while, <br><input class="submitbutton_ss simulator-btn grafana-btn-red" name="Once" value="Submit Once">: will only generate data once. 
+                    <br><br>If you select <input class="submitbutton_temp simulator-btn grafana-btn-green" name="Continuous" value="Continuous Submission"> , 
+                    click on <input class="submitbutton_temp simulator-btn grafana-btn-red" name="Stop" value="Stop Submission"> to discontinue generating data. 
+                    <br><br>
+                    To generate data for all panels at once, you will find buttons for all as shown below: <br> <input class="submitbutton_all grafana-btn grafana-btn-green" name="Continuous" value="Continuous Submission All">
+                    
+                    <input class="submitbutton_all grafana-btn grafana-btn-red" name="Once" value="Submit All Once">
+                    
+                    <input class="submitbutton_all grafana-btn grafana-btn-red" name="Stop" value="Stop All Submission"> <br> in the center. 
```

Comment:
Can you add the `readonly` attribute to each of the `<input>` elements? They are currently modifiable, but shouldn't be since they are just description. Also, I don't think the `name` parameter does anything here, so it could be removed form each element. 

As an example, we could change
`<input class="submitbutton_all grafana-btn grafana-btn-red" name="Stop" value="Stop All Submission">`
to
`<input class="submitbutton_all grafana-btn grafana-btn-red" value="Stop All Submission" readonly>`